web/stripe-template/api/routes/webhook/POST-product-price.js

```diff
@@ -0,0 +1,196 @@
+import { RouteContext } from "gadget-server";
+import { stripe, getStripeWebhookEvent } from "../../stripe";
+import { destructure, objKeyConvert } from "../../utils";
+
+// Change all to use background actions
+
+/**
+ * Route handler for POST webhook/product-price
+ *
+ * @param { RouteContext } route context - see: https://docs.gadget.dev/guides/http-routes/route-configuration#route-context
+ *
+ */
+export default async function route({ request, reply, api, logger }) {
+  let event = request.body;
+
+  try {
+    event = getStripeWebhookEvent({
+      request,
+      endpointSecret: process.env.STRIPE_PRODUCT_PRODUCT_PRICE_WEBHOOK_SECRET,
+    });
+    logger.info({ event }, "my event");
+  } catch (err) {
+    logger.error({ err }, "Stripe webhook error");
+    return await reply.status(400).send();
+  }
+
+  // handle webhook events
+  // all webhooks fire corresponding create, update, or delete actions for the "product" and "price" models
+  switch (event.type) {
+    case "price.created":
+      const priceCreated = destructure({
+        topic: "price",
+        obj: objKeyConvert(event.data.object),
+      });
```

Comment:
I hate how I did this, and we have the same repeated destructure everywhere. Is it possible we move this to the `stripe` client file?

Reply:
I'm going to remove price all together